gh_pages/_source/session5/Building-a-Perception-Pipeline.rst

```diff
@@ -135,6 +135,7 @@ Viewing Results
 
       cd ~
       roscore
+      rosrun tf static_transform_publisher 0 0 0 0 0 0 world_frame kinect_link 100
```

Comment:
Suggestion: check whether STP from `tf2_ros` works. That will result in a significant reduction in bw and CPU usage.